frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -337,3 +356,57 @@ func validateRequest(r *apiv1.CreateCVDRequest) error {
 	}
 	return nil
 }
+
+// See https://pkg.go.dev/io/ioutil@go1.13.15#TempFile
+func createTempFile(pattern string, data []byte, mode os.FileMode) (*os.File, error) {
+	file, err := ioutil.TempFile("", pattern)
+	if err != nil {
+		return nil, err
+	}
+	if err := file.Chmod(mode); err != nil {
+		return nil, err
+	}
+	_, err = file.Write(data)
+	if closeErr := file.Close(); closeErr != nil && err == nil {
+		err = closeErr
+	}
+	if err != nil {
+		return nil, err
+	}
+	return file, nil
+}
+
+func createCredsFile(ctx cvd.CVDExecContext) (string, error) {
+	name, err := tempFilename("cvdload*.creds")
+	if err != nil {
+		return "", err
+	}
+	if err := cvd.Exec(ctx, "touch", name); err != nil {
+		return "", err
+	}
+	if err := cvd.Exec(ctx, "chgrp", "cvdnetwork", name); err != nil {
```

Comment:
why does `cvdnetwork` need write access to the credentials file?

Reply:
this way we can write in the credentials from the `host_orchestrator` process run by the `_cutf-operator`.

Reply:
The HO may or may not run in a single user environment. Allowing any type of access to the `cvdnetwork` group is risky, even it's only for writing.
If possible, create the file with the operator owner, write it and then change the ownership. That would keep access only in the users that should have it.
Or at least remove the group access after writing if the above is not possible.

Reply:
I see, I tried the `change ownership` approach but `chown` resulted in `operation not permitted` in this case. Implemented the temporary write permission approach.